__tests__/app.test.js

```diff
@@ -125,6 +201,16 @@ describe("GET /api/articles", () => {
 
         });
     });
+    test("order error, value is not allowed (order only)", () => {
+      return request(app)
+        .get("/api/articles?order=dasc")
+        .expect(400)
+        .then(({ body }) => {
+
+          expect(body.msg).toBe("Invalid query");
+
+        });
+    });
```

Comment:
Nice start to error handling, but we also need error handling tests for:

When given an invalid sort_by:
E.g `/api/articles?sort_by=banana"`